Contributing.md

```diff
@@ -0,0 +1,69 @@
+# Contributing
+When contributing to this repository, please first discuss the change you wish to make via issue, email, or any other method with the owners of this repository before making a change.
+Please note we have a code of conduct, please follow it in all your interactions with the project.
+
+# Pull Request Process
+1.	Fork the repository and create a separate branch regarding your contribution.
+2.	Update the README.md with details of changes to the interface, this includes new environment variables, exposed ports, useful file locations, and container parameters.
+3.	Issue a pull request and ask us to review it. Don't forget to link the PR to a relevant issue if it exists.
+4.	Make sure the pull request passes all checks
+5.	You may merge the Pull Request once you have the sign-off of two other developers, or if you do not have permission to do that, you may request the second reviewer to merge it for you.
+
+
+# Issues
+### Solve an issue
+Scan through our existing issues to find one that interests you. You can narrow down the search using labels as filters. See Labels for more information. We assign issues to specific contributors, although you are more than welcome to open a PR with a fix if you find an issue to work on.
+
+### Create a new issue
+If you spot a problem with the docs, search if an issue already exists. If a related issue doesn't exist, you can open a new issue using a relevant issue form.
```

Comment:
only relevant to a problem with the docs? the issues are used for every change\bug\feature\feat needed.